scripts/install-dependencies.sh

```diff
@@ -88,3 +88,9 @@ install_aar "com.google.android.gms" "play-services" "6.5.87" "${GOOGLE_REPOSITO
 
 # Install maps
 install_maps "com.google.android.maps" "maps" "18" "4"
+
+# Update the Android jar with layoutlib.jar
+mkdir /tmp/robolectric-dependencies.layoutlib21
+cd /tmp/robolectric-dependencies.layoutlib21; jar xf ${ANDROID_HOME}/platforms/android-21/data/layoutlib.jar; rm android/view/accessibility/AccessibilityManager*
+jar uf ~/.m2/repository/org/robolectric/android-all/5.0.0_r2-robolectric-1/android-all-5.0.0_r2-robolectric-1.jar . 
```

Comment:
Right now, I am overriding the classes in the android-all jar with classes in the layout jar, I am not sure how to include both sets of classes and load them accordingly. Perhaps this is another way to work around the resource management issue and make other tests pass.